src/Groups/sub.jl

```diff
@@ -92,6 +85,11 @@ end
 
 (f::GAPGroupHomomorphism)(x::GAPGroupElem) = image(f, x)
 
+"""
+    image(f::GAPGroupHomomorphism, x::GAPGroupElem)
+    (f::GAPGroupHomomorphism)(x::GAPGroupElem)
+Return `f`(`x`).
```

Comment:
```suggestion
Return the image of `x` under the homomorphism `f`. Can also be written as `f(x)` or `x^f`.
```
I am not sure that `x^f` is currently possible, but for GAP user's that'd be nice to enable, so perhaps you can add it. In GAP, homomorphisms are applied from the right, after all, just like permutations and everything else.

Reply:
Done.